tests/ui/error-codes/E0388.stderr

```diff
@@ -29,20 +29,8 @@ LL | static STATIC_REF: &'static mut i32 = &mut X;
    = help: add `#![feature(const_mut_refs)]` to the crate attributes to enable
    = note: this compiler was built on YYYY-MM-DD; consider upgrading it if it is out of date
 
-error[E0764]: mutable references are not allowed in the final value of statics
-  --> $DIR/E0388.rs:6:39
-   |
-LL | static STATIC_REF: &'static mut i32 = &mut X;
-   |                                       ^^^^^^
-
-error[E0596]: cannot borrow immutable static item `X` as mutable
-  --> $DIR/E0388.rs:6:39
```

Comment:
I don't know why this one disappears, I assume it is some sort of "do not show more errors if there are already other errors" logic -- there's no way my patch prevents the borrow checker from noticing this `&mut X` of an immutable static, right?

Reply:
does something weird happen like the `&mut X` getting promoted?

Reply:
`X` is an `i32` so I don't think so.

I didn't investigate this since I don't think this is the load-bearing test for "cannot borrow immutable static item `X` as mutable". For instance, this is still in tests/ui/borrowck/mutability-errors.rs.